WalletWasabi/Legal/LegalDocuments.cs

```diff
@@ -10,63 +10,84 @@ namespace WalletWasabi.Legal
 {
 	public class LegalDocuments
 	{
-		public const string EmbeddedFileName = "LegalDocuments.txt";
-		public const string LegalFolderName = "Legal";
-		public const string AssetsFoldername = "Assets";
-		public static readonly string EmbeddedFilePath = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), LegalFolderName, AssetsFoldername, EmbeddedFileName);
+		public static readonly string EmbeddedFilePath = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), "Legal", "Assets", "LegalDocuments.txt");
 
-		public LegalDocuments(string filePath)
+		public LegalDocuments(Version version, string content)
```

Comment:
It's actually very confusing to have `LegalDocuments` class if it represents a single document (at least `string content` makes me think so). If it is so, I would go with `LegalDocument`.

Reply:
I would actually split this class into two classes. One for holding data:

`public LegalDocument(Version version, string content)`

and one class (`LegalDocumentsUtils`) for static methods that operate with `LegalDocument` instances.

Reply:
> It's actually very confusing to have `LegalDocuments` class if it represents a single document (at least `string content` makes me think so). If it is so, I would go with `LegalDocument` or some other name in English-singular noun.

I agree but I won't do it - or won't do it here. It is a huge change - I think we can live with it.

Reply:
> I would actually split this class into two classes. One for holding data:
> 
> `public LegalDocument(Version version, string content)`
> 
> and one class (`LegalDocumentsUtils`) for static methods that operate with `LegalDocument` instances.

The class is not big enough to split it - it is readable IMO. I prefer it this way, at least we have all in the same place.